scripts/parser-tests/typescript/allowlist.txt

```diff
@@ -398,7 +398,7 @@ shorthandPropertyAssignmentInES6Module.ts
 sourceMap-LineBreaks.ts
 sourceMapValidationDecorators.ts
 sourceMapValidationStatements.ts
-staticIndexer.ts
+staticModifierAlreadySeen.ts
```

Comment:
We should bump the TS commit hash (41dc625b0a609eb294b975dd92675e72b2b3fdec) in the `makefile` to make sure the related tests are run.